mlflow/langchain/__init__.py

```diff
@@ -918,7 +918,7 @@ def _load_model_from_local_fs(local_model_path, model_config_overrides=None):
             # We would like to clean up the dependencies schema which is set to global
             # after loading the mode to avoid the schema being used in the next model loading
             _clear_dependencies_schemas()
-
+        _add_code_from_conf_to_system_path(local_model_path, flavor_conf)
```

Comment:
This line should be moved to the place before L914.

Reply:
fixed!